integration-tests/instructions-with-bytes-conversion.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { CONFIGS } from "./config";
+import { Protocols } from "@taquito/taquito";
+import { bytesAndInt, bytesAndNat } from "./data/instructions-bytes-conversions-contracts";
+import { HttpResponseError } from "@taquito/http-utils";
+
+CONFIGS().forEach(({ lib, protocol, setup }) => {
+  const Tezos = lib;
+  const limanet = protocol === Protocols.PtLimaPtL ? test : test.skip;
+  const mumbaiAndAlpha = protocol === Protocols.PtMumbaii || protocol === Protocols.ProtoALpha ? test : test.skip;
+
+  describe(`Test origination of contract with instructions now supporting bytes`, () => {
+
+    beforeEach(async (done) => {
+      await setup();
+      done();
+    });
+
+    mumbaiAndAlpha(`Should be able to orignate contract with ADD parameter in michelson contract with bytes`, async done => {
```

Comment:
typo originate*

Reply:
Thanks for the comment never changed these test names!